stock_view_adj_hls/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock View Adj HLS",
+    "summary": "",
+    "description": """
+    """,
+    'version': '12.0.1.0.0',
+    "category": "Stock",
+    "website": "https://www.quartile.co/",
+    "author": "Quartile Limited",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+         'stock_report_secondary_unit',
```

Comment:
Why do we need this dependency?

Reply:
we need it to use the field removal_date.